osf/models/contributor.py

```diff
@@ -35,11 +27,22 @@ def bibliographic(self):
 
     @property
     def permission(self):
```

Comment:
Now that nodes are using guardian, PreprintContributor model can probably inherit from Node contributor instead of redefining everything.